### DISCUSSION TITLE
how can i check double entry and block them

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#204] [Q&A] [answered]
I swear is saw it in the documentation but not find it.
If some one has already booket to a Event so how can i check that he already has bookt the event by press submitting. User all are not regestried on the page

**Comment by liedekef** [comment, marked as answer]
This is a RSVP-setting when creating an event: "Allow only 1 booking per unique email address" or "Allow only 1 booking per person (combo email/last name/first name)"

**Reply by loelismurf** [reply]
Yeah thanks it works, but is it not possible in the general section i see right?

**Reply by liedekef** [reply]
No, this is a setting per event.

**Reply by loelismurf** [reply]
Okey thanks for the answer, would be nice if we can set it in the Settings Page for default value are turn on :=)
